core/scale/libp2p_types.hpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_SCALE_LIBP2P_TYPES_HPP
+#define KAGOME_CORE_SCALE_LIBP2P_TYPES_HPP
+
+#include <algorithm>
+#include <exception>
```

Comment:
Move it to .cpp, please

Reply:
true, I just forgot to do it. Previously the whole implementation was put into the header file only.